packages/state-transition/src/cache/epochCache.ts

```diff
@@ -187,11 +206,13 @@ export class EpochCache {
   // TODO: Helper stats
   epoch: Epoch;
   syncPeriod: SyncPeriod;
+  isAfterEIP6110: boolean;
```

Comment:
Having to persist this value looks like a premature optimization. Doing a number equality check is really cheap. Just define a new class method if you want to de-duplicate code.